internal/server/routes.go

```diff
@@ -135,6 +130,11 @@ func (s *Server) GenerateRoutes() Routes {
 
 	add(a, noAuthnWithOrg, http.MethodGet, "/.well-known/jwks.json", wellKnownJWKsRoute)
 
+	// Device flow
+	post(a, noAuthnNoOrg, "/api/device", a.StartDeviceFlow)
+	post(a, noAuthnWithOrg, "/api/device/status", a.GetDeviceFlowStatus)
```

Comment:
I think we could use the org_id from either the user or provider that we have IDs for, right? I'm not sure if that's necessary for this PR, but it seems like we don't necessarily need to get the org from the request domain.

However it would require relaxing our requirement that `data.GetIdentity` sets an orgID in the transaction metadata. We'd need an option to `GetIdentity` to tell it to skip the org requirement.

Reply:
Indeed. Went with this for now given `GetIdentity` requires the org id in the transaction